frontend/src/metabase/api/user.ts

```diff
@@ -0,0 +1,100 @@
+import type {
+  CreateUserRequest,
+  ListUsersRequest,
+  ListUsersResponse,
+  UpdatePasswordRequest,
+  UserId,
+  User,
+  UpdateUserRequest,
+} from "metabase-types/api";
+
+import { Api } from "./api";
+import {
+  idTag,
+  listTag,
+  invalidateTags,
+  provideUserTags,
+  provideUserListTags,
+} from "./tags";
+
+export const userApi = Api.injectEndpoints({
+  endpoints: builder => ({
+    listUsers: builder.query<ListUsersResponse, ListUsersRequest | void>({
+      query: body => ({
+        method: "GET",
+        url: "/api/user",
+        body,
+      }),
+      providesTags: response =>
+        response?.data ? provideUserListTags(response.data) : [],
```

Comment:
```suggestion
        response ? provideUserListTags(response.data) : [],
```

Reply:
Nope!
There were 10 different FE unit tests that were failing.

`response` may or may not contain `data`.

Reply:
> `response` may or may not contain `data`.

In that case either `ListUsersResponse` typing is incorrect or mocks are incorrectly set up in unit tests.
We should fix that.